.github/workflows/qc.yml

```diff
@@ -27,10 +27,35 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Run ontology QC checks
+        id: check
+        continue-on-error: true
         env:
           DEFAULT_BRANCH: master
           ROBOT_JAVA_ARGS: -Xmx6G
         run: cd src/ontology && mkdir -p tmp reports mirror && make ROBOT_ENV='ROBOT_JAVA_ARGS=-Xmx6G' GH_ACTION=true BRI=false MIR=false IMP=false test > TESTLOG.log
+
+      - name: Reason over taxon constraints
+        id: explaintc
+        continue-on-error: true
+        if: steps.check.outcome == 'failure'
```

Comment:
This check could go on another yml file.

Reply:
Doing it in another workflow would mean that the taxon constraint check should _also_ be performed, again, in that workflow. That’d be a waste of resources since the check has already been done here as part of the main QC workflow.

Reply:
ok